docs/index.rst

```diff
@@ -5,18 +5,81 @@
 Welcome to scikit-fda's documentation!
 ======================================
 
+This package offers classes, methods and functions to give support to
+Functional Data Analysis in Python. Includes a wide range of utils to work with
+functional data, and its representation, exploratory analysis, or
+preprocessing, among other tasks such as inference, classification, regression
+or clustering of functional data.
+
+In the `project page <https://github.com/GAA-UAM/scikit-fda>`_ hosted by
+Github you can find more information related to the development of the package.
+
+
 .. toctree::
-   :includehidden:
-   :maxdepth: 4
+   :maxdepth: 2
    :caption: Contents:
    :titlesonly:
 
    apilist
+
+
+.. toctree::
+   :maxdepth: 1
+   :titlesonly:
+
    auto_examples/index
 
-Indices and tables
-==================
+An exhaustive list of all the contents of the package can be found in the
+:ref:`genindex`.
+
+Installation
+------------
+
+Currently, scikit-fda is available in Python 3.6 and 3.7, regardless of the
+platform. The stable version can be installed via
+`PyPI <https://pypi.org/project/scikit-fda/>`_:
+
+.. code-block:: bash
+
+   pip install scikit-fda
+
+
+It is possible to install the latest version of the package, available in
+the develop branch, by cloning this repository and doing a manual installation.
+
+.. code-block:: bash
+
+   git clone https://github.com/GAA-UAM/scikit-fda.git
+   cd scikit-fda/
+   pip install -r requirements.txt
```

Comment:
This is not really needed.

Reply:
The ``pip install -r requirements.txt`` command or the section with the manual installation?

Reply:
The `requirements.txt` are not necessary.

Reply:
It was placed by the dependencies of the ``setup.py``
https://github.com/GAA-UAM/scikit-fda/blob/e81577b5b13aafa6b1c4aa00f8c90dee4bfe11a3/setup.py#L26-L31

Reply:
I think that  having the `pyproject.toml` suffices. I have never used the `requirements.txt` for installing.

Reply:
mmm, in that case, should it be installed with pip instead of running ``python setup.py``?

```bash
 git clone https://github.com/GAA-UAM/scikit-fda.git
 pip install ./scikit-fda
```

Reply:
I think so.

Reply:
Done in 11f9081.